### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Ray2126**

### CODE REVIEW

File: src/components/navbar/NavBar.stories.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 
 import { Auth0Provider } from "../../utils/react-auth0-spa";
-import Navbar from "./Navbar";
+import NavBar from "./Navbar";
 
-storiesOf("Navbar", module)
+storiesOf("NavBar", module)
   .addDecorator((storyFn) => <Auth0Provider>{storyFn()}</Auth0Provider>)
-  .add("default view", () => <Navbar />);
```
Comment (webber-jiang) [inline src/components/navbar/NavBar.stories.js:9]: You might also need to addDecorator react router.

File: src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -1,26 +1,169 @@
...
+        style={{ marginTop: 5 }}
+      >
+        <Dropdown.Menu>
+          <Dropdown.Item text="New" />
+        </Dropdown.Menu>
+      </Dropdown>
+    );
+  };
+
+  return (
```
Comment (webber-jiang) [inline src/components/navbar/Navbar.jsx:150]: Try use ternary here.

**Comment by webber-jiang** [comment]
Now that the project is in TS, this will also have to be in TS

**Comment by Ray2126** [comment]
Can you approve this? I will change it to Typescript after it is merged with the master branch as I need the configuration for typescript files so it will run.

**Pull request closed without merging**